src/ast-to-react.js

```diff
@@ -77,7 +77,8 @@ exports.hastChildrenToReact = childrenToReact
  *
  * @typedef {keyof IntrinsicElements} ReactMarkdownNames
  *
- * @typedef {Object.<string, unknown>} ReactBaseProps
+ * @typedef {Object} ReactBaseProps
+ * @property {string?} className
```

Comment:
```suggestion
 * @typedef {{ [key: string]: unknown, className?: string }} ReactBaseProps
```

https://www.typescriptlang.org/play?useJavaScript=true#code/PQKhCgAIUgBAXAngBwKYBNUDNIG9eQDaA1qogFyQDO8ATgJYB2A5gLqUCujxjA9gO6MANJADGAGwCGVKgDlJAW1QB+SjQYtIAXy2QAogA9Fycaighg4cKAjQ4SNHkPHTW85dG9GNSPFQ+AXjwtIA

Reply:
@kikagaku You can click “commit suggestion” on these suggestions to add then. I just did that for you!